interfaces/builtin/basedeclaration_test.go

```diff
@@ -345,29 +345,30 @@ var (
 
 	slotInstallation = map[string][]string{
 		// other
-		"bluez":            {"app"},
-		"bool-file":        {"core", "gadget"},
-		"browser-support":  {"core"},
-		"content":          {"app", "gadget"},
-		"dbus":             {"app"},
-		"docker-support":   {"core"},
-		"fwupd":            {"app"},
-		"gpio":             {"core", "gadget"},
-		"hidraw":           {"core", "gadget"},
-		"i2c":              {"core", "gadget"},
-		"iio":              {"core", "gadget"},
-		"location-control": {"app"},
-		"location-observe": {"app"},
-		"lxd-support":      {"core"},
-		"mir":              {"app"},
-		"modem-manager":    {"app", "core"},
-		"mpris":            {"app"},
-		"network-manager":  {"app", "core"},
-		"ofono":            {"app", "core"},
-		"ppp":              {"core"},
-		"pulseaudio":       {"app", "core"},
-		"serial-port":      {"core", "gadget"},
-		"udisks2":          {"app"},
+		"bluez":             {"app"},
+		"bool-file":         {"core", "gadget"},
+		"browser-support":   {"core"},
+		"classic-dimension": {"core"},
+		"content":           {"app", "gadget"},
+		"dbus":              {"app"},
+		"docker-support":    {"core"},
+		"fwupd":             {"app"},
+		"gpio":              {"core", "gadget"},
+		"hidraw":            {"core", "gadget"},
+		"i2c":               {"core", "gadget"},
+		"iio":               {"core", "gadget"},
+		"location-control":  {"app"},
+		"location-observe":  {"app"},
+		"lxd-support":       {"core"},
+		"mir":               {"app"},
+		"modem-manager":     {"app", "core"},
+		"mpris":             {"app"},
+		"network-manager":   {"app", "core"},
+		"ofono":             {"app", "core"},
+		"ppp":               {"core"},
+		"pulseaudio":        {"app", "core"},
+		"serial-port":       {"core", "gadget"},
+		"udisks2":           {"app"},
```

Comment:
That was all done by `go fmt` - I just added the `classic-dimension: {"core"}` change.